lib/composite_cache_store.rb

```diff
@@ -4,120 +4,132 @@
 require_relative "composite_cache_store/version"
 
 class CompositeCacheStore
-  DEFAULT_OUTER_OPTIONS = {
+  DEFAULT_LAYER_1_OPTIONS = {
     expires_in: 5.minutes,
     size: 16.megabytes
   }
 
-  DEFAULT_INNER_OPTIONS = {
+  DEFAULT_LAYER_2_OPTIONS = {
     expires_in: 1.day,
     size: 32.megabytes
   }
 
-  attr_reader :outer_cache_store, :inner_cache_store
-
-  alias_method :outer, :outer_cache_store
-  alias_method :inner, :inner_cache_store
+  attr_reader :layers
+  delegate_missing_to :"layers.last"
```

Comment:
for some reason, this doesn't work (yet):

```
@store.normalized_key("asdf")
NoMethodError: undefined method `normalized_key' for #<CompositeCacheStore:0x00000001063c44c8 @layers=[#<ActiveSupport::Cache::MemoryStore entries=0, size=0, options={:expires_in=>1 second, :size=>8388608, :compress=>false, :compress_threshold=>1024}>, #<ActiveSupport::Cache::MemoryStore entries=0, size=0, options={:expires_in=>1 minute, :size=>67108864, :compress=>false, :compress_threshold=>1024}>]>
from /Users/jrubisch/.rbenv/versions/3.1.1/lib/ruby/gems/3.1.0/gems/activesupport-7.0.4.2/lib/active_support/core_ext/module/delegation.rb:307:in `method_missing'
```

Reply:
I'm removing the delegation on this PR. We can follow up with another one to re-add if it makes sense.